indico/modules/events/editing/notifications.py

```diff
@@ -0,0 +1,96 @@
+# This file is part of Indico.
+# Copyright (C) 2002 - 2020 CERN
+#
+# Indico is free software; you can redistribute it and/or
+# modify it under the terms of the MIT License; see the
+# LICENSE file for more details.
+
+from indico.core.notifications import email_sender, make_email
+from indico.web.flask.templating import get_template_module
+from indico.web.flask.util import url_for
+
+
+def _get_timeline_url(revision):
+    editable = revision.editable
+    contribution = editable.contribution
+    return url_for('event_editing.editable', contribution.event, contribution, type=editable.type.name, _external=True)
+
+
+@email_sender
+def notify_submitter_comment(revision):
+    """
+    Notifies the editor about comments made by a submitter.
+    """
+    editor = revision.editable.editor
+    subject = 'New submitter comment'
```

Comment:
I am wondering whether the subjects should be passed like that to the templates. As far as I can tell each template has a different subject thus you could probably put those strings directly in the `txt` files.

Reply:
Of course, still using the `block subject` since it is being used in the `get_subject` macro

Reply:
Yes, that's exactly what the subject block is meant for :)